.ci.yaml

```diff
@@ -4622,3 +4622,22 @@ targets:
       tags: >
         ["devicelab", "hostonly", "mac"]
       task_name: flutter_tool_startup__macos
+
+  # This is used to test the new release scheduler. This will eventually be deleted.
```

Comment:
Is this testable following: go/flutter-luci-recipes#testing-a-local-change-for-a-new-builderrecipe?

Reply:
It should be. This is only meant to be temporary, but it's a duplicate of the current target: https://github.com/flutter/flutter/blob/master/.ci.yaml#L302